test/unit/query/builder.js

```diff
@@ -6023,6 +6023,34 @@ describe('QueryBuilder', function() {
     );
   });
 
```

Comment:
I went ahead and created an integration test that uses two tables, locking only rows from one of them.

Reply:
@tgriesser problem of not having integration tests is that no one ever tests that given queries are actually doing what they are supposed to do. There has been many bugs because of that. So integration tests are not testing that DB works correctly, but it makes sure that created query does what it is supposed to do.